docs/installation-and-operation/serialization.md

```diff
@@ -12,116 +12,364 @@ Once you really get rolling with Metabase, it's often the case that you'll have
+
+By default, Metabase includes some database settings and the data model in the export. Database connection strings are _not_ included by default, unless you [explicitly ask Metabase to include them](#customize-what-gets-exported). You can also opt out of including the data model entirely.
+
+Metabase serializes databases and tables in the `databases` directory. It will include YAML files for every database, table, field, segment, and metric.
+
+Databases, tables, and fields are referred to by their names (unlike Metabase-specific items, which are [referred to by entity IDs](#metabase-uses-entity-ids-to-identify-and-reference-metabase-items)).
+
+For example, in the [Example of a serialized question](#example-of-a-serialized-question), there are several YAML keys that reference Sample Database:
+
+```yaml
+database_id: Sample Database
+---
+dataset_query:
+  database: Sample Database
+```
+
+In the description of the field filter (`category_filter:`) in that example, you can see the reference to the field that's used to populate the filter options:
+
+```yaml
+dimension:
+  - field
+  - - Sample Database
+    - PUBLIC
+    - PRODUCTS
+    - CATEGORY
+```
+
+It refers to the `CATEGORY` field in the `PRODUCTS` table in the `PUBLIC` schema in `Sample Database`. The serialized `Sample Database` in the `databases` directory will also include YAML files for this field and table.
+
+## How import works
+
+During import, Metabase will read the provided YAML files and create items according to the YAML specs. [Example of a serialized question](#example-of-a-serialized-question) how Metabase records information it needs to reconstruct an item.
+
+Metabase will not delete items from target instance during import, but it will overwrite items that already exist.
+
+Metabase relies on [Entity IDs](#metabase-uses-entity-ids-to-identify-and-reference-items) to figure out which items to create or overwrite, and what are the relationships between items. When importing into an instance that already has some content in it, keep in mind:
+
+- If you import an item with an `entity_id` that already exists in your target Metabase, the existing item will be overwritten.
+
+  In particular, this means that if you export a question, then make a change in an exported YAML file — like rename a question by directly editing the `name` field — and then import the edited file back, Metabase will try to apply the changes you made to the YAML.
+
+- If you import an item with blank `entity_id` (and blank `serdes/meta → id`), Metabase will create a new item.
+
+- All items and data sources referenced in YAML must either exist in the target Metabase already, or be included in the import.
+
+  For example, if an exported YAML has the field `collection_id: onou5H28Wvy3kWnjxxdKQ`, then the collection `onou5H28Wvy3kWnjxxdKQ` must already exist in target instance, or there must be a YAML file with the export of a collection that has this ID.
+
+## Serialization best practices
```

Comment:
I think it's more like "things to keep in mind", but I couldn't come up with a good title